features/exploredash/src/main/java/org/dash/wallet/features/exploredash/data/model/merchent/GetDataMerchantIdResponse.kt

```diff
@@ -0,0 +1,507 @@
+                val totalMilliseconds: Int? = 0,
+                @SerializedName("TotalMinutes")
+                val totalMinutes: Int? = 0,
+                @SerializedName("TotalSeconds")
+                val totalSeconds: Int? = 0
+            )
+
+            data class WednesdayOpen(
+                @SerializedName("Days")
+                val days: Int? = 0,
+                @SerializedName("Hours")
+                val hours: Int? = 0,
+                @SerializedName("Milliseconds")
+                val milliseconds: Int? = 0,
+                @SerializedName("Minutes")
+                val minutes: Int? = 0,
+                @SerializedName("Seconds")
+                val seconds: Int? = 0,
+                @SerializedName("Ticks")
+                val ticks: Int? = 0,
+                @SerializedName("TotalDays")
+                val totalDays: Int? = 0,
+                @SerializedName("TotalHours")
+                val totalHours: Int? = 0,
+                @SerializedName("TotalMilliseconds")
+                val totalMilliseconds: Int? = 0,
+                @SerializedName("TotalMinutes")
+                val totalMinutes: Int? = 0,
+                @SerializedName("TotalSeconds")
+                val totalSeconds: Int? = 0
+            )
+        }
+
+        data class Merchant(
+            @SerializedName("AcceptsTips")
+            val acceptsTips: Boolean? = false,
+            @SerializedName("CardImageUrl")
+            val cardImageUrl: String? = "",
+            @SerializedName("CardholderAgreement")
+            val cardholderAgreement: String? = "",
+            @SerializedName("Description")
+            val description: String? = "",
+            @SerializedName("HasBarcode")
+            val hasBarcode: Boolean? = false,
+            @SerializedName("Id")
+            val id: Int? = 0,
+            @SerializedName("IsOnline")
+            val isOnline: Boolean? = false,
+            @SerializedName("IsPhysical")
+            val isPhysical: Boolean? = false,
+            @SerializedName("IsVariablePurchase")
+            val isVariablePurchase: Boolean? = false,
+            @SerializedName("LegalName")
+            val legalName: String? = "",
+            @SerializedName("LogoUrl")
+            val logoUrl: String? = "",
+            @SerializedName("MaximumCardPurchase")
+            val maximumCardPurchase: Int? = 0,
+            @SerializedName("MinimumCardPurchase")
+            val minimumCardPurchase: Int? = 0,
+            @SerializedName("PaymentInstructions")
+            val paymentInstructions: String? = "",
+            @SerializedName("SavingsPercentage")
+            val savingsPercentage: Int? = 0,
+            @SerializedName("SystemName")
+            val systemName: String? = "",
+            @SerializedName("TermsAndConditions")
+            val termsAndConditions: String? = "",
+            @SerializedName("Website")
+            val website: String? = ""
+        )
+    }
+}
```

Comment:
Do we need all of these or just a couple of classes/fields? Having the entirety of the response in here might be confusing for someone dealing with this code later.

Reply:
I add this api for @HashEngineering  to use in his ticket

Reply:
I understand that, but there are a lot of fields here and we might only need several. In his PR @HashEngineering has a much smaller amount of fields: https://github.com/dashpay/dash-wallet/blob/16560031887d19f1d85020875601aff6261efcae/features/exploredash/src/main/java/org/dash/wallet/features/exploredash/data/model/merchants/GetMerchantByIdResponse.kt

Reply:
In this PR, we can leave all of the fields as they are.  In my PR, I will remove what is not needed and make a few other necessary changes to the data types on the required fields.